packages/language-server/src/plugins/svelte/features/getCompletions.ts

```diff
@@ -28,13 +41,40 @@ export function getCompletions(
     const notPreceededByOpeningBracket = !/[\s\S]*{\s*[#:/@]\w*$/.test(
         lastCharactersBeforePosition,
     );
-    if (isInStyleOrScript || notPreceededByOpeningBracket) {
+    if (isInStyleOrScript) {
         return null;
     }
 
-    const triggerCharacter = getTriggerCharacter(lastCharactersBeforePosition);
-    // return all, filtering with regards to user input will be done client side
-    return getCompletionsWithRegardToTriggerCharacter(triggerCharacter, svelteDoc, offset);
+    if (notPreceededByOpeningBracket) {
+        return getComponentDocumentationCompletions();
+    }
+
+    return getTagCompletionsWithinMoustache();
+
+    /**
+     * Get completions for special svelte tags within moustache tags.
+     */
+    function getTagCompletionsWithinMoustache() {
+        const triggerCharacter = getTriggerCharacter(lastCharactersBeforePosition);
+        // return all, filtering with regards to user input will be done client side
+        return getCompletionsWithRegardToTriggerCharacter(triggerCharacter, svelteDoc, offset);
+    }
+
+    function getComponentDocumentationCompletions() {
+        if (!lastCharactersBeforePosition.includes(HTML_COMMENT_START)) {
+            return null;
+        }
+
+        const commentStartIndex = lastCharactersBeforePosition.lastIndexOf(HTML_COMMENT_START);
+        const text = lastCharactersBeforePosition.substring(
+            commentStartIndex + HTML_COMMENT_START.length
+        ).trimLeft();
```

Comment:
Maybe also newlines could also be trimmed here, so that 
```
<!--
@ 
```
will get the autocompletion, too

Reply:
Seems like it does trim new line. Doesn't know that.

Reply:
Oh nice ^^